addons/binding/org.openhab.binding.seneye/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="seneye"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <thing-type id="seneye">
+        <label>Seneye Monitor</label>
+        <description>Seneye Monitor Thing</description>
+
+        <channels>
+            <channel id="channeltemperature" typeId="seneye-temperature-channel"/>
+            <channel id="channelph" typeId="seneye-ph-channel"/>
+            <channel id="channelnh3" typeId="seneye-nh3-channel"/>
+            <channel id="channelnh4" typeId="seneye-nh4-channel"/>
+            <channel id="channelO2" typeId="seneye-O2-channel"/>
+            <channel id="channellux" typeId="seneye-lux-channel"/>
+            <channel id="channelpar" typeId="seneye-par-channel"/>
+            <channel id="channelkelvin" typeId="seneye-Kelvin-channel"/>
+            <channel id="channellastreading" typeId="seneye-lastreading-channel"/>
+            <channel id="channelslideexpires" typeId="seneye-slideexpires-channel"/>
+        </channels>
+        <config-description>
+            <parameter name="aquariumname" type="text" required="true">
+                <label>Aquarium Name</label>
+                <description>The name of your aquarium</description>
+                <advanced>false</advanced>
+                <required>true</required>
+            </parameter>
+
+            <parameter name="username" type="text" required="true">
+                <label>Username</label>
+                <description>The username of My Seneye</description>
+                <advanced>false</advanced>
+                <required>true</required>
+            </parameter>
+
+            <parameter name="password" type="text" required="true">
+                <label>Password</label>
+                <description>The password of My Seneye</description>
+                <context>password</context>
+                <advanced>false</advanced>
+                <required>true</required>
+            </parameter>
+            
+            <parameter name="polltime" type="integer" required="true" min="1" max="1440">
+                <label>Polling time</label>
+                <description>How often (in minutes) does the seneye needs to be checked ?</description>
+                <default>5</default>
+                <advanced>false</advanced>
+                <required>true</required>
+            </parameter>
+
+        </config-description>        
+    </thing-type>
+
+    <!-- Channels -->
+    <channel-type id="seneye-temperature-channel">
+        <item-type>String</item-type>

```

Comment:
Why did you choose type `String` here? The values that you are passing to the channels are all numbers, right? so how about changing this to type `Number`?

The bonus you will get with a `Number` channel is support for units such as celsius, lux, etc. This is currently work in progress but almost finished: https://github.com/eclipse/smarthome/pull/1857 So if you use `Number` channels it will be easy for you to use that once it is finished.

Reply:
thanks for pointing out that Quantity type, this will be an improvement once it's merged into openhab.
Changed the type to number